src/key_io.cpp

```diff
@@ -370,6 +370,19 @@ std::string KeyIO::EncodePaymentAddress(const libzcash::PaymentAddress& zaddr) c
     return std::visit(PaymentAddressEncoder(keyConstants), zaddr);
 }
 
+std::string KeyIO::EncodeTexAddress(const CKeyID& p2pkhAddr) const
+{
+    CDataStream ss(SER_NETWORK, PROTOCOL_VERSION);
+    ss << p2pkhAddr;
+    // ConvertBits requires unsigned char, but CDataStream uses char
+    std::vector<unsigned char> seraddr(ss.begin(), ss.end());
+    std::vector<unsigned char> data;
+    // See calculation comment below
```

Comment:
Copy-pasta, needs updating:
```suggestion
    // See calculation comment above
```